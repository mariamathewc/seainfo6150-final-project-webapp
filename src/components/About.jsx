import React from "react";
import styles from "./About.module.css";
import b1 from "../images/business1.jpg"
import b2 from "../images/business4.jpg"
import b3 from "../images/business3.jpg"
const About = (props) => {




    return (
    <div>

        <div className={styles.aboutsection}>
                <h1><em>CONNECTING GENERATIONS TO THE OUTDOORS</em></h1>
            
        </div>

            <h2 className={styles.heading} >Our Team</h2>

        <div className={styles.row}>
            <div className={styles.column}>
                    <div className={styles.card}>
                        <img src={b1} alt="John" className={styles.imageIcon} />
                        <div className={styles.container}>
                            <h2>John Doe</h2>
                            
                            <p>Email: <a href="mailto:john@example.com">john@example.com</a></p>
                            
                            <br />

                            <p><button className={styles.button}>CEO & Founder</button></p>
                        </div>
                </div>
            </div>

            <div className={styles.column}>
                <div className={styles.card}>
                        <img src={b2} alt="Mike" className={styles.imageIcon}/>
                            <div className={styles.container}>
                                <h2>Mike Ross</h2>
                            
                            <p>Email: <a href="mailto:mike@example.com">mike@example.com</a></p>
                            <br />
                            <p><button className={styles.button}>Vice President</button></p>
                            </div>
                </div>
            </div>

            <div className={styles.column}>
                <div className={styles.card}>
                        <img src={b3} alt="Jane" className={styles.imageIcon}/>
                            <div className={styles.container}>
                                <h2>Jane Doe</h2>
                           
                            <p>Email: <a href="mailto:jane@example.com">jane@example.com</a></p>
                            <br />
                            <p><button className={styles.button}>Sales Manager</button></p>
                            </div>
                </div>
            </div>
        </div>
    </div>


    );


}

export default About