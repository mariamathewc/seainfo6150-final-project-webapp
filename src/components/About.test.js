import React from "react";
import About from "./About.jsx";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';



describe("About test", () => {
	it("renders correctly", () => {
		const { container } = render(<BrowserRouter><About /></BrowserRouter>);
		expect(container).toMatchSnapshot();
	});
});