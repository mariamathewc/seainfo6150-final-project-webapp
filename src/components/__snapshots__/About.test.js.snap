// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About test renders correctly 1`] = `
<div>
  <div>
    <div
      class="aboutsection"
    >
      <h1>
        <em>
          CONNECTING GENERATIONS TO THE OUTDOORS
        </em>
      </h1>
    </div>
    <h2
      class="heading"
    >
      Our Team
    </h2>
    <div
      class="row"
    >
      <div
        class="column"
      >
        <div
          class="card"
        >
          <img
            alt="John"
            class="imageIcon"
            src="business1.jpg"
          />
          <div
            class="container"
          >
            <h2>
              John Doe
            </h2>
            <p>
              Email: 
              <a
                href="mailto:john@example.com"
              >
                john@example.com
              </a>
            </p>
            <br />
            <p>
              <button
                class="button"
              >
                CEO & Founder
              </button>
            </p>
          </div>
        </div>
      </div>
      <div
        class="column"
      >
        <div
          class="card"
        >
          <img
            alt="Mike"
            class="imageIcon"
            src="business4.jpg"
          />
          <div
            class="container"
          >
            <h2>
              Mike Ross
            </h2>
            <p>
              Email: 
              <a
                href="mailto:mike@example.com"
              >
                mike@example.com
              </a>
            </p>
            <br />
            <p>
              <button
                class="button"
              >
                Vice President
              </button>
            </p>
          </div>
        </div>
      </div>
      <div
        class="column"
      >
        <div
          class="card"
        >
          <img
            alt="Jane"
            class="imageIcon"
            src="business3.jpg"
          />
          <div
            class="container"
          >
            <h2>
              Jane Doe
            </h2>
            <p>
              Email: 
              <a
                href="mailto:jane@example.com"
              >
                jane@example.com
              </a>
            </p>
            <br />
            <p>
              <button
                class="button"
              >
                Sales Manager
              </button>
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
